app/views/simulator/edit.html.erb

```diff
@@ -4,295 +4,302 @@
-    </button>
-    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-      <ul class="navbar-nav mr-auto noSelect pointerCursor" id="toolbar">
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project<span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li class="toolbarButton logixButton" id="newProject" ><a>New Project</a></li>
-            <li class="toolbarButton logixButton" id="clearProject" ><a>Clear Project</a></li>
-            <li class="toolbarButton logixButton" id="save"><a>Save Online</a></li>
-            <li class="toolbarButton logixButton" id="saveOffline"><a>Save Offline</a></li>
-            <li class="toolbarButton logixButton" id="createOpenLocalPrompt"><a>Open Offline</a></li>
-            <li class="toolbarButton logixButton" id="recoverProject"><a>Recover project</a></li>
-          </ul>
-        </li>
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Circuit<span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li class="toolbarButton logixButton" id="newCircuit"><a>New Circuit +</a></li>
-            <li class="toolbarButton logixButton" id="createSubCircuitPrompt"><a>Insert SubCircuit</a></li>
-          </ul>
-        </li>
+<script>
+  userSignedIn = <%= user_signed_in? %>;
+  logix_project_id = <%= @logix_project_id %>;
+  projectName = "<%= @projectName %>"
+</script>
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="/">CircuitVerse</a>
+  <button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-controls="collapsedNavbar" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+    <ul class="navbar-nav mr-auto noSelect pointerCursor" id="toolbar">
+      <li class="dropdown">
+        <a href="#" class="dropdown-toggle text-light" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project<span class="caret"></span></a>
+        <ul class="dropdown-menu">
+          <li><a class="dropdown-item" id="newProject">New Project</a></li>
+          <li><a class="dropdown-item" id="clearProject">Clear Project</a></li>
+          <li><a class="dropdown-item" id="save">Save Online</a></li>
+          <li><a class="dropdown-item" id="saveOffline">Save Offline</a></li>
+          <li><a class="dropdown-item" id="createOpenLocalPrompt">Open Offline</a></li>
+          <% if user_signed_in? %>
+          <li class="toolbarButton logixButton" id="createOpenOnlinePrompt">
+                <a href="<%= user_path(current_user) %>">Open Online</a>
```

Comment:
@Shivansh2407 actually, this is not the `open online` that was intended. Basically, we need a way to access the project page of the project. Rather it should be `view project page`.  see `project_path`

Reply:
Also why are there so many changes in this PR, ideally you just be adding a link to the drop down.

Reply:
Actually I have added link and did some changes too in data.js file but afterwards only change remaining was in the edit.html.erb file. I will be fixing this issue soon .